src/script/assets/AssetCrypto.js

```diff
@@ -17,21 +17,18 @@
  *
  */
 
-window.z = window.z || {};
-window.z.assets = z.assets || {};
-
-z.assets.AssetCrypto = (() => {
+class AssetCrypto {
```

Comment:
I think a class is not the right abstraction here, it is too powerful for what we want to achieve. I would go for exporting the 2 public functions with the 
```
export const decrypt.. = () => {}
```

thus making the other functions private by design